tutorial-server/notebooks/CactusTutorial.ipynb

```diff
@@ -763,21 +781,55 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# store output directory location for later use\n",
-    "# in ipython you can also use this:\n",
-    "# outdir = ! ./simfactory/bin/sim get-output-dir tov_ET\n",
-    "# outdir = outdir[0]\n",
+    "from IPython.display import Image\n",
+    "\n",
+    "# The following is only to find the path of the simulation output.\n",
+    "# See comment in the previous cell. We add this here to keep this\n",
+    "# tutorial general.\n",
     "import os\n",
-    "outdir = os.popen(\"./simfactory/bin/sim get-output-dir tov_ET\").read().rstrip(\"\\n\")\n",
+    "outdir = os.popen(\"./simfactory/bin/sim get-output-dir tov_ET\").read().rstrip(\"\\n\")[:-12]\n",
     "\n",
-    "print (\"Output is written to\", outdir)"
+    "Image(filename=outdir+\"/rho_maximum.png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or we can plot a 2D slice of the rest-mass density with `plot_grid_var.py` (which does not support reflection symmetry, see below). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Here we define a bash variable that contains the path where\n",
+    "# the simulation data lives. We do so to keep this tutorial general.\n",
+    "# Normally you would just input the path or you would run the\n",
+    "# script from the output directory.\n",
+    "datadir=$(dirname $(./simfactory/bin/sim get-output-dir tov_ET))\n",
+    "./utils/Analysis/kuibit/examples/bins/plot_grid_var.py --datadir $datadir \\\n",
```

Comment:
I'd add some text that explains what `x0` and `x1` are (lower left and uppper right corner?).